### PR TITLE
Update pools.json to reflect the actual file

### PR DESCRIPTION
**Pull request by max1ne**
The previous version before this is not reflecting the actual file on server. It has been maintained and updated frequently off Github for a while.  Herein I replace the content of this pools.json with the actual file on all the existing ZEL Explorers!

### CODE REVIEW

**Comment by TheTrunk** [comment]
@max1ne 
On our servers there are 2 pools missing but are present in this repository. Please reflect the PR to be in correspondence with Equihup #2 and with Equihash.pro https://github.com/zelcash/insight-api/commit/05da98b1725a3347a1a215589aca0ed84c13a560#diff-e20a9f7abfd0a7151b56854fd4cf2f8fR67
Thank you for a PR

File: pools.json [hunk 1]
```diff
@@ -50,39 +29,17 @@
...
         "searchStrings": [
             "t1VoiZk4sdvzstMbDTVgYDzBgBEL3dCPuUZ",
             "t1fp4WH4QL3zJa3zeFFu2PcT34tbMma5HUc",
-            "t1bkSfToeGPMjnm6AvEhjGZdFvRZvksD4SX",
-            "t1LFTQYatpcPLqmkoW9kDL1ZmQK2jSqMSfk"
+            "t1bkSfToeGPMjnm6AvEhjGZdFvRZvksD4SX"
         ]
     },
     {
-        "poolName": "Equihub",
```
Comment (TheTrunk) [inline pools.json:71]: need to be reflected according to #2

File: pools.json [hunk 2]
```diff
@@ -50,39 +29,17 @@
...
-        ]
-    },
     {
         "poolName": "Equihash.pro",
         "url": "https://equihash.pro/",
         "searchStrings": [
             "t1VoiZk4sdvzstMbDTVgYDzBgBEL3dCPuUZ",
             "t1fp4WH4QL3zJa3zeFFu2PcT34tbMma5HUc",
-            "t1bkSfToeGPMjnm6AvEhjGZdFvRZvksD4SX",
-            "t1LFTQYatpcPLqmkoW9kDL1ZmQK2jSqMSfk"
```
Comment (TheTrunk) [inline pools.json:67]: The last one deleted is currently correct pool address
https://github.com/zelcash/insight-api/commit/05da98b1725a3347a1a215589aca0ed84c13a560#diff-e20a9f7abfd0a7151b56854fd4cf2f8fR67

**Review by TheTrunk — CHANGES_REQUESTED** [review]

**Comment by max1ne** [comment]
Latest Equihash.Pro has been incorporated on the last lines.
I added one missing for EquiHash.Pro
To not confuse this, I'll close this PR and issue a fresh new one.

**Pull request closed without merging**